src/shared/components/ExternalTools/ExternalTool.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
```

Comment:
all of this stuff I think belongs in the external configuration itself.  we should just allow people to add a button with a callback and the business logic of the specific tool belongs in that callback, outside of the cbio codebase